blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -24,15 +24,15 @@ import { calculateNode } from '../util/KaraokeHelper';
 
 const { string, object, any, func } = PropTypes;
 
-const queuedState = () => (
+const QueuedState = () => (
     <EmptyStateView tightSpacing>
         <p>
             <Icon {...{
                 size: 20,
                 icon: 'timer',
                 style: { fill: '#fff' },
             }} />
-            <span>Waiting for run to start.</span>
```

Comment:
It worked with freestyle (and other job types that have a real queued state). We had a ticket around 2 weeks ago where we covered that part. However the queued state for that is ATM broken.

I am now reusing this and added a hook for AT to be able to identified the content.